cmstestsuite/testrunner.py

```diff
@@ -100,13 +101,24 @@ def load_cms_conf():
         return get_cms_config()
 
     def log_elapsed_time(self):
+        def duration_to_human(start, end):
+            millis = int((end - start).total_seconds() * 1000)
```

Comment:
Tip: you could use `str(end - start)` which returns a similar format (though not exactly identical) to save yourself some effort.

Reply:
Ah, I was looking for something like that -,-